### DISCUSSION TITLE
Litte update in your code

### DISCUSSION THREAD

**Discussion by gui00francisco** [Jarvis-Ank/Re-Vanced#6] [Ideas]
I changed a few things in your code to make it even easier. And I added a cell to export the results to Drive.
It's my first time programming. Was it good?

https://colab.research.google.com/drive/1x_twA_ft2kARrm33TikAnWMIK04ALAeq

**Comment by Jarvis-Ank** [comment]
It surely is good. Mounting the gdrive was omitted because some people don't clearly know how this works and they don't want to give permission to drive even though it's safe. Otherwise the step to add [file_id] could be removed and automated to fetch from drive directly. I'm happy that you are able to change the code for your needs, even i use same mounting method.
